src/main/java/org/sagebionetworks/web/client/widget/entity/renderer/HtmlPreviewWidget.java

```diff
@@ -43,7 +43,7 @@ public class HtmlPreviewWidget implements IsWidget, HtmlPreviewView.Presenter {
 	protected PopupUtilsView popupUtils;
 	protected GWTWrapper gwt;
 
-	public static final double MAX_HTML_FILE_SIZE = 40 * MB;
+	public static final double MAX_HTML_FILE_SIZE = 10 * MB;
```

Comment:
The max user-provided html file size was set to 5MB in PreviewWidget, and 40MB here.  What do you think about increasing the limit from 5MB to 10MB?  10MB is a pretty large html file (so it may take some time to download and parse), but I don't think this would make Synapse.org look terrible.

Reply:
Sounds good.